### ISSUE TITLE
Dimmed dialogs and accessibility

### ISSUE THREAD

**Issue by droidery** [SwiftKickMobile/SwiftMessages#66]
When using VoiceOver, users can still select views behind a dimmed message. To reproduce:

1. On an iOS device, enable VoiceOver: Settings > General > Accessibility > VoiceOver > On
2. Open demo app with Taco View (dimmed with `config.dimMode = .gray(interactive: true)`)
3. Select views in background, as highlighted here
<img width="461" alt="screen shot" src="https://cloud.githubusercontent.com/assets/25188027/22400866/cbbd3202-e614-11e6-8d01-d042e016ea53.png">

Since you can do this, users can navigate all around the app without realizing the message to order tacos should be completed.

Without VoiceOver, users cannot select views behind the dimmed message until they deal with the order tacos dialog (this is the expected behavior).

A solution to improve accessibility for low vision users would not to announce things when there's a selection in dimmed area. This would prevent navigation without the user completing a dimmed dialog.

**Other info**
I found this property in the Apple accessibility documentation: `accessibilityElementsHidden`. The description outlines same issue that I've noticed: 

> You might use this property to hide views that are covered by the arrival of a new view. In this case, the hidden views might remain visible onscreen, but they are not the focus of the user’s actions.

**Comment by wtmoose** [comment]
Good suggestion. I'll look into this as soon as I can free up some time.

**Comment by wtmoose** [comment]
Fixed in 3.2.0. Thanks!

**Comment by wtmoose** [comment]
FYI, I just published [3.0.0 with improved VoiceOver support](https://github.com/SwiftKickMobile/SwiftMessages/releases/tag/3.3.0).

**Comment by gcharita** [comment]
@wtmoose this issue seems to reappearing in later versions. Tested in iOS 15.6.1 and I still can select the views behind dimmed dialogs.

**Comment by gcharita** [comment]
Never mind. Changing the `presentationContext` to `.window(windowLevel: .alert)` or `.window(windowLevel: .statusBar)` fixes this issue.
It seems that `accessibilityViewIsModal` is ignored by the system if window level is `.normal`.
